### ISSUE TITLE
XAML Binding dropdown inserts extra characters

### ISSUE THREAD

**Issue by tonyfonager** [mfractor/mfractor-feedback#304]
When I edit inside a XAML document and want to add a Binding value to a property, a fine dropdown selector comes up.

But when I select values from this dropdown, they are inserted twice.

![image](https://user-images.githubusercontent.com/837894/98943348-a214fc00-24ef-11eb-848c-6f33c7b9d0fd.png)

Makes it insert this

![image](https://user-images.githubusercontent.com/837894/98943405-b78a2600-24ef-11eb-8fee-fba364125303.png)

If I after the dropdown appear, starts typing (to find the correct property), instead of inserting it twice it inserts it once + the remainder of what I typed.

So for the same dropdown as above, I start typing to find the property "IsVIP" - here I have types "isv"

![image](https://user-images.githubusercontent.com/837894/98943562-fcae5800-24ef-11eb-8afe-c25e8381e1fb.png)

And now it inserts this

![image](https://user-images.githubusercontent.com/837894/98943621-1354af00-24f0-11eb-9e04-5356ced3ddd7.png)

So basically it inserts the selected binding property + the selected binding property MINUS what you might have typed already to narrow down the list of possibilities!

I have also attached todays logfiles, to help you investigate 👍 

[log.zip](https://github.com/mfractor/mfractor-feedback/files/5530742/log.zip)

##Installation Information

```
--Platform Information--
Operating System: Mac
OS: Unix
OS Service Pack: 
OS Version: Unix 19.6.0.0
--MFractor Information--
Product Name: MFractor
Product Version: 4.3.3.0
Product Variant: Visual Studio Mac
Product Variant Version: 8.8.0.2913
Product SKU: MFRACTOR-VS-MAC
Commit SHA: 18c0d29f339bf4321ded3a59430f48f3019e459d
Revision: 4597
Branch: master
Date: 24/09/2020 1:07:23 AM (UTC)
Agent: Local
Installed extensions:	'DeepClean (1.2.5)', 'Xamarin Hot Reload (1.0.0.2968096)', 'MFractor (4.3.3)', 'XML Editor (8.8)', 'Makefile generation (8.8)', 'Visual Designer Support (8.8)', 'MonoDevelop Text Editor (8.8)', 'Version Control Support (8.8)', 'Android development (8.8)', 'Xamarin Ide Extensions for Mac (8.8)', 'Azure Support (8.8)', 'ASP.NET Core Support (8.8)', 'Refactoring Support (8.8)', 'Razor Language Services (8.0.20201013.7)', 'Xamarin Ide Extensions (8.8)', 'Text Templating (8.8)', '.NET Core Support (8.8)', 'NuGet Packaging (8.8)', 'Performance Diagnostics of IDE (8.8)', 'Xamarin Test Cloud (8.8)', 'Xamarin Test Cloud - NUnit (8.8)', 'MonoDevelop Source Editor (8.8)', 'Test Runner (8.8)', 'Regex Toolkit (8.8)', 'JavaScript Debugger (8.8)', 'Xamarin Designer (16.8.0.495)', 'Subversion support (8.8)', 'Web Project Support (8.8)', 'CSharp Language Binding (8.8)', 'MonoDevelop Text Editor (8.8)', 'Xamarin Hot Reload (1.0.0.4127551)', 'GTK# Visual Designer (8.8)', 'MonoDevelop Assembly Browser (8.8)', 'Android Deploy (16.8.0.495)', 'MonoDevelop Hex Editor (8.8)', 'Xamarin Ide Identity (8.8)', 'NuGet Package Management (8.8)', 'Web Editors (8.3)', 'Mono Soft Debugger Support (8.8)', 'Xamarin Addins Core (8.8)', 'Xamarin Terminal (8.8)', 'Debugger support for MonoDevelop (8.8)', 'Container Tools (1.0.20200521.1)', 'Xamarin.Mac development (8.8)', 'VisualFSharp (8.8)', 'GDB Debugger Support (8.8)', 'ConnectedServices (8.8)', 'JavaScript and TypeScript (8.3)', 'MonoDevelop Mac Platform Support (8.8)', 'Git support (8.8)', 'Xamarin.Forms Previewer (16.8.0.495)', 'ILAsm Language Binding (8.8)', 'Android Designer (16.8.0.495)', 'Project Web References (8.8)', 'Subversion core engine (8.8)', 'Forms Project Support (8.8)', 'NUnit support (8.8)', 'Mac Development (8.8)', 'Unit Testing core support (8.8)', 'iOS development (8.8)', 'Visual Studio for Mac Tools for Unity (2.8.3.0)', 'Gettext Translations Support (8.8)', 'Azure Mobile App Service (8.8)', 'Azure DevOps Authentication Support (8.8)', 'Deployment Services Core (8.8)', '.Net Core Debugger (8.8)', 'Azure Functions development (8.8)', 'Extension Developer Tools (8.8)', 'Mono Soft Debugger Support for ASP.NET (8.8)', 'VsCode Debug Protocol support for MonoDevelop (8.8)', 'XamlLanguageService (16.8.0.495)', 'iOS Designer (16.8.0.495)', 'ChangeLog Add-in (8.8)', 'VB.NET Language Binding (8.8)'
```

**Comment by matthewrdev** [comment]
@tonyfonager Thanks for reporting this issue.

This has been fixed and a build will be available later today (16th of November).

**Comment by matthewrdev** [comment]
@tonyfonager This is now fixed and released in MFractor 4.4.
